src/views/template/factory/factory.html

```diff
@@ -8,6 +8,20 @@
         {{#l}}Furthermore, a shipping charge of{{/l}} <strong id="order-transport-cost"></strong>
         {{#l}}is always incurred, regardless of order size.{{/l}}
     </div>
+    <div class="panel-footer">
```

Comment:
The `panel-footer` should be below the `panel-body`, as that makes more sense. Beyond that, I think this looks fine.

Do make sure you remove the `data-customer` attributes from the buttons!